scripts/apps/packaging/services/PackagesService.ts

```diff
@@ -1,5 +1,13 @@
 import _ from 'lodash';
 import {AuthoringWorkspaceService} from 'apps/authoring/authoring/services/AuthoringWorkspaceService';
+import {gettext} from 'core/utils';
+
+function getRefLabel(ref) {
```

Comment:
```
gettext(ref);
        ^ only static strings should be passed to gettext
```


Aside from that, the group object and label are still separate. It could be united by defining the main group like this:

`const mainGroup = {idRef: 'main', label: gettext('Main')};`

This way, there's no `default` switch condition and we are can be certain that the string is translated and that no dead translations will remain if objects are removed from the code.

Reply:
oops I meant to put `gettext('main')`, my bad.